dlf/plugins/search/flexform.xml

```diff
@@ -240,6 +251,21 @@
 							</config>
 						</TCEforms>
 					</targetPid>
+					<targetPidPageView>
+						<TCEforms>
+							<displayCond>FIELD:showSingleResult:=:0</displayCond>
```

Comment:
This option should be shown when `showSingleResult` is set to `true`, right? Then its `displayCond` should be `FIELD:showSingleResult:=:1`.

Reply:
No, it's the other way round. The change to "Show single result in pageview" fixes this, too.